Formula/coreos-ct.rb

```diff
@@ -0,0 +1,59 @@
+class CoreosCt < Formula
+  desc "Convert a Container Linux Config into Ignition"
+  homepage "https://coreos.com/os/docs/latest/configuration.html"
+  url "https://github.com/coreos/container-linux-config-transpiler/archive/v0.5.0.tar.gz"
+  sha256 "172e44796d39ec117584e121e73194cbcb56701407badc1899cea4607a343e15"
+
+  depends_on "go" => :build
+
+  def install
+    system "git", "init"
```

Comment:
We don't run `system "git"` inside our install. This should be built directly from the released tarball.

Reply:
There usually isn't a release tarball for `go` stuff, at least in source format. Could use a `git` checkout and `tag` as the URL though if commit/tag information is absolutely necessary here.